### PR TITLE
Upgrade mjolnir.js to v3

### PR DESCRIPTION
**Pull request by Pessimistress**
#### Change List
- Upgrade mjolnir.js to v3, remove postinstall hack script that fixes hammer.js
- Update Deck class to use v3 API - mjolnir's EventManager no longer comes with preset recognizers, they need to be explicitly defined
- `tripan` event is changed to `multipan` and requires two fingers instead of three
- Docs & upgrade guide

### CODE REVIEW

File: modules/core/src/lib/constants.ts [hunk 1]
```diff
@@ -101,12 +108,27 @@ export const UNIT = {
 } as const;
 
 export const EVENTS = {
-  click: {handler: 'onClick'},
+  tap: {handler: 'onClick'},
```
Comment (felixpalmer) [inline modules/core/src/lib/constants.ts:111]: This is a very common event. Could changing it break apps?

**Review by felixpalmer — APPROVED** [review]

File: modules/core/src/lib/constants.ts [hunk 2]
```diff
@@ -101,12 +108,27 @@ export const UNIT = {
 } as const;
 
 export const EVENTS = {
```
Comment (ibgreen) [inline modules/core/src/lib/constants.ts:110]: Can these map declarations be properly typed now with mjolnir 3? Maybe an opportunity to make a typing pass?

**Review by ibgreen — APPROVED** [review]

File: modules/core/src/lib/constants.ts (hunk 2 above)
Comment (Pessimistress) [reply modules/core/src/lib/constants.ts:110]: Not sure what types we need for this particular object. It is only used by the Deck class and doesn't have anything to do with mjolnir.

Comment (ibgreen) [reply modules/core/src/lib/constants.ts:110]: IMHO, we shouldn't have untyped objects in our code base.

File: modules/core/src/lib/constants.ts (hunk 1 above)
Comment (Pessimistress) [reply modules/core/src/lib/constants.ts:111]: In the previous version, mjolnir defined preset recognizers and some "aliases", including tap <-> click, doubletap <-> dblclick

This is no longer the case, so I guess we want to pick one that breaks less apps?

`eventManager` is private and apps are not supposed to attach their own event listener. A custom Controller class has the option to listen to additional events, so in theory some will see the breaking change, but I don't imagine tap/click is utilized by many.

**Pull request merged**